tests/regression/diffkemp/clean_test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```

Comment:
Isn't this file just moved? Shouldn't it be removed from the last location as well?

Reply:
Yes, the file was moved, and this was correctly detected by Git. It shows it here as being newly created because it was detected as being moved to tests/regression/diffkabi (which can be seen [here](https://github.com/viktormalik/diffkemp/pull/15/commits/bd9b04342eccbd6249be9470a8ebb997f3dc0a5d)), therefore the copy in tests/regression/diffkemp is seen as a new file. The same thing happened with regression_test.py.

Reply:
Ah, I see, now we have two of these files.